src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -14,30 +14,637 @@
  * limitations under the License.
  */
 
-export const human38stats = {
-  assembly_stats: {
-    chromosome_count: 23,
-    genome_size: 3096649726,
-    total_sequence_length: 4537931177
+export enum AllStatsSections {
+  CODING_STATS = 'coding_stats',
+  NON_CODING_STATS = 'non_coding_stats',
+  PSEUDOGENES = 'pseudogene_stats',
+  ASSEMBLY_STATS = 'assembly_stats',
+  ASSEMBLY_SUMMARY = 'assembly_summary'
+}
+
+type RawSpeciesStats = {
+  [genomeId: string]: {
+    [key in AllStatsSections]: {
+      [key: string]: string | number | null;
+    };
+  };
+};
+
+export const sampleData: RawSpeciesStats = {
+  homo_sapiens_GCA_000001405_28: {
+    assembly_summary: {
+      scientific_name: 'Homo sapiens',
+      sex: null,
+      strain: null,
+      taxonomy_id: 9606,
+      assembly_name: 'GRCh38.p13',
+      assembly_accession: 'GCA_000001405.28',
+      assembly_date: '2019-02-28'
+    },
+    assembly_stats: {
+      contig_n50: 56413054,
+      total_genome_length: 3272116950,
+      total_coding_sequence_length: 34317174,
+      total_gap_length: 161368351,
+      spanned_gaps: 661,
+      chromosomes: 25,
+      toplevel_sequences: 640,
+      component_sequences: 36734,
+      gc_percentage: 38.87
+    },
+    coding_stats: {
+      scientific_name: 'Homo sapiens',
```

Comment:
This has been moved to assembly summary. Same for other sections and other species below.

Reply:
Remove it

Reply:
I'm still seeing it in multiple places in the file (https://github.com/Ensembl/ensembl-client/blob/ac354582bcc65b250a2e298b70d189246e16669f/src/ensembl/src/content/app/species/sample-data.ts).

Reply:
Should be gone now?